### PR TITLE
Portuguese translation README.adoc file examples/xa-datasource

### PR DESCRIPTION
**Pull request by jrxxjr**
Please validate the translation.

### CODE REVIEW

File: examples/xa-datasource/README.adoc [hunk 1]
```diff
@@ -0,0 +1,331 @@
+Title: Injection Of Entitymanager
```
Comment (Daniel-Dos) [inline examples/xa-datasource/README.adoc:1]: add the following header:

:index-group: DataSources
:jbake-type: page
:jbake-status: status=published
= Injection Of Entitymanager

File: examples/xa-datasource/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,334 @@
+Title: Injeção do EntityManager
```
Comment (Daniel-Dos) [inline examples/xa-datasource/README_pt.adoc:1]: add the following header:

:index-group: DataSources
:jbake-type: page
:jbake-status: status=published
= Injection Of Entitymanager

File: examples/xa-datasource/README.adoc (hunk 1 above)
Comment (jrxxjr) [reply examples/xa-datasource/README.adoc:5]: Is corrected

Comment (Daniel-Dos) [reply examples/xa-datasource/README.adoc:5]: sorry , please remove tihis -> Title: Injection Of Entitymanager

File: examples/xa-datasource/README_pt.adoc (hunk 2 above)
Comment (jrxxjr) [reply examples/xa-datasource/README_pt.adoc:6]: Is corrected

File: examples/xa-datasource/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,339 @@
+:index-group: DataSources
+:jbake-type: page
+:jbake-status: status=published
+= Injection Of Entitymanager
+
+Title: Injeção do EntityManager
```
Comment (Daniel-Dos) [inline examples/xa-datasource/README_pt.adoc:6]: Sorry , please remove this -> Title: Injection Of Entitymanager

**Comment by jrxxjr** [comment]
The title stay in english = Injection Of Entitymanager, or translated to portuguese = Injeção do EntityManager, because this file is of the portuguese translation ?

**Comment by jrxxjr** [comment]
The title stay in english = Injection Of Entitymanager, or translated to portuguese = Injeção do EntityManager, because this file is of the portuguese translation ?

**Comment by Daniel-Dos** [comment]
@jrxxjr can translate : )

File: examples/xa-datasource/README_pt.adoc (hunk 3 above)
Comment (Daniel-Dos) [inline examples/xa-datasource/README_pt.adoc:6]: remove

File: examples/xa-datasource/README_pt.adoc (hunk 2 above)
Comment (Daniel-Dos) [reply examples/xa-datasource/README_pt.adoc:6]: please remove this -> Title: Injeção do EntityManager

Comment (Daniel-Dos) [reply examples/xa-datasource/README_pt.adoc:6]: and fix   = Injection Of Entitymanager to =  Injeção do EntityManager

**Comment by jrxxjr** [comment]
Then will stay like this, follows below:   
:index-group: DataSources
:jbake-type: page
:jbake-status: status=published
= Injeção do EntityManager

**Comment by Daniel-Dos** [comment]
yes 

:index-group: DataSources
:jbake-type: page
:jbake-status: status=published
= Injeção do EntityManager

**Comment by jrxxjr** [comment]
Is corrected

**Comment by Daniel-Dos** [comment]
thanks @jrxxjr

**Pull request merged**